src/vs/editor/browser/widget/diffEditorWidget2/diffEditorEditors.ts

```diff
@@ -99,7 +99,7 @@ export class DiffEditorEditors extends Disposable {
 			// Disable unicode highlighting for the original side in inline mode, as they are not shown anyway.
 			result.unicodeHighlight = { nonBasicASCII: false, ambiguousCharacters: false, invisibleCharacters: false };
 		} else {
-			result.unicodeHighlight = this._options.editorOptions.get().unicodeHighlight;
+			result.unicodeHighlight = this._options.editorOptions.get().unicodeHighlight || {};
```

Comment:
feels weird to make this `unicodeHighlight` an optional property yet initialize it with an empty object.

Reply:
Yes, this is not nice. I have to take a closer look in debt week.